Gruntfile.coffee

```diff
@@ -116,6 +116,22 @@ module.exports = (grunt) ->
           dest: '.tmp/js/payload.js'
         ]
 
+      plugins_flavor:
+        options:
+          patterns: [
+            {
+              match: 'flavor'
+              replacement: -> grunt.config.get('current_flavor')
```

Comment:
This is equivalent to:
```
replacement: "<%= grunt.config.get('current_flavor') %>"
```

Templating has also been used [here](https://github.com/skroutz/analytics.js/blob/build_flavors/Gruntfile.coffee#L355); perhaps we should remain consistent and use a single approach.